lightningd/peer_control.c

```diff
@@ -705,10 +705,13 @@ static void json_add_peer(struct lightningd *ld,
 		if (channel->owner)
 			json_add_string(response, "owner",
 					channel->owner->name);
-		if (channel->scid)
+		if (channel->scid) {
 			json_add_short_channel_id(response,
 						  "short_channel_id",
 						  channel->scid);
+			json_add_num(response, "channel_direction",
```

Comment:
We've always called this just `direction` and given the context it's redundant.

Reply:
I don't think we've exposed in via JSON before, but I can s/channel_/ everywere.